client/me/security-checkup/recovery-email-validation-notice.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+
+class RecoveryEmailValidationNotice extends Component {
```

Comment:
Is there or will there be a `RecoveryPhoneValidationNotice` components? Could they be combined into one with a `type` prop or similar?